### ISSUE TITLE
WCAG Failure: When selecting the thumbnail or next page, focus does not move to the top of the new pages

### ISSUE THREAD

**Issue by patdunlavey** [ProjectMirador/mirador#3920]
This is from an external accessibility audit of our Mirador 3 install.
- Failed WCAG Success Criteria: 1.3.2 Meaningful Sequence (A)
- Description: When selecting the thumbnail or next page, focus does not move to the top of the new pages. Users have to navigate back up the page to read the content.
- Recommendation: Move focus to the top of the new content.

**Comment by marlo-longley** [comment]
Hi @patdunlavey not sure if you are still following this issue but I just took a look, thanks for reporting.

I wasn't exactly clear on what part of the app you are referring to here. I am attaching a screen recording of where I think the failure occurs. Tab focus seems to work with thumbnails until a click selection is made in the bottom section. Then the focus returns to the beginning of the page. I believe you are saying the focus should remain in the thumbnail section. Thanks for any insight!

https://github.com/user-attachments/assets/3d279f4e-786c-419b-b3c6-a7f5ec38ca82
